style.css

```diff
@@ -172,8 +172,7 @@ button {
   font-size: 17px;
   line-height: 24px;
   border: 1px solid hsla(234, 100%, 69%, 1);
-  width: 124px;
-  height: 48px;
+  padding: 6px 12px;
   border-radius: 8px;
   background-color: white;
   color: #396df2;
```

Comment:
- [ ] Please increase the contrast ratio between the background color and text color in hover state for all `see project` buttons because this ratio supposes to be >= 4.5 to enhance the site accessibility.

![image](https://user-images.githubusercontent.com/68796232/187466525-831e885a-dbb9-478e-aab0-88153d5b5f6f.png)

### [Hint] You can use this [tool ](https://www.aditus.io/button-contrast-checker/)to test your fix.